vendor/bat-native-ads/src/bat/ads/internal/country_subdivision.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_COUNTRY_SUBDIVISION_H_
+#define BAT_ADS_INTERNAL_COUNTRY_SUBDIVISION_H_
+
+#include <stdint.h>
+#include <string>
+#include <map>
+#include <memory>
```

Comment:
is this needed?

Reply:
changed https://github.com/brave/brave-core/pull/5602/files#diff-2b8dd5c95bb138082aa12e6833cee528R9